src/runtime/auto-prompt-manager.ts

```diff
@@ -167,6 +161,13 @@ export interface ShowAutoPromptParams {
   isClosable?: boolean;
 }
 
+interface LocalStorageTimestamps {
```

Comment:
nit: what do you think about renaming this to `ActionTimestamps` to make it clear that it's per action type?